### ISSUE TITLE
[Bug][a11y] Homescreen cannot be accessed through talkback gestures

### ISSUE THREAD

**Issue by softvision-miralobontiu** [mozilla-mobile/fenix#9404]
## Steps to reproduce
1. Turn TalkBack (or the available screen reader) on.
2. Navigate the homescreen buttons and fields using talkback gestures/swipe. Make sure all items can be reached/read.
3. Make sure all items have a clear description.

### Expected behavior
All items are read and have clear descriptions.

### Actual behavior
The focus is on the private browsing icon. Swiping does nothing. Only tapping on each item moves the focus, and only then the item is read or described.

### Device information
* Android devices: OnePlus 5T (Android 9), Sony Xperia Z5 Premium (Android 7.1.1)
* Fenix version: Beta 4.2.0-beta.2, Nightly from 3/25
* Please see the attached logcat
* Reproducible on the private browsing homescreen

[logcat.txt](https://github.com/mozilla-mobile/fenix/files/4381682/logcat.txt)

**Comment by asadotzler** [comment]
I'm unable to reproduce this on a Nightly build from 3/28. With Talkback enabled I can swipe through all of the controls of the Fenix homescreen.

**Comment by softvision-miralobontiu** [comment]
I was not able to reproduce the issue anymore, using OnePlus 5T (Android 9), and Sony Xperia Z5 Premium (Android 7.1.1), with the latest Nightly build from 4/1.
I'll close the issue.
